backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/lsm/LiveMigrateDiskCommand.java

```diff
@@ -657,6 +657,20 @@ protected boolean validateDestDomainsSpaceRequirements() {
         return true;
     }
 
+    protected StorageDomainValidator getDestStorageDomainValidator(StorageDomain destStorageDomain) {
```

Comment:
looks like the only change that's needed and looks fine to me

Reply:
@barpavel how about isolating this change and separating out the refactoring to a different PR?

Reply:
@ahadas, Yes, already started

Reply:
Done.